ambry-api/src/main/java/com.github.ambry/config/HelixAccountServiceConfig.java

```diff
@@ -54,12 +57,28 @@
   @Default("")
   public final String backupDir;
 
+  /**
+   * If true, then use the new znode path to store list of blob ids that point to account metadata content.
+   */
+  @Config(USE_NEW_ZNODE_PATH)
+  @Default("false")
+  public final boolean useNewZNodePath;
+
+  /**
+   * The number of latest versions of account metadata to save in ambry.
+   */
+  @Config(VERSION_COUNT)
+  @Default("10")
+  public final int versionCount;
```

Comment:
better to rename this, current name doesn't quite match the comment.  How about `maxNumOfVersionsInMem` or `maxNumOfCachedVersions`?

Reply:
I will remove this option for this PR since it's logic is not yet implemented. Will revisit it at next PR.